tests/integration_tests/util.py

```diff
@@ -168,7 +168,11 @@ def append_or_create_list(
             ignore_warnings,
             "Unable to disable SSH logins for opc given ssh_redirect_user",
         )
-
+    # Preserve platform-specific tracebacks expected
```

Comment:
So was traceback_texts just not being used before when it was supposed to? If `None` or a list is passed as ignore_tracebacks, should we just use that use and append to that rather than keeping a separate list here?

Reply:
If False is provided, we want to report any tracebacks. If None is provided, that's the default and in default case I think we want to ignore expected tracebacks per platform.  If a list is passed then we should append to that list instead of keeping the separate list here.